src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -170,6 +170,8 @@
 	@ModElementReference public List<BiomeEntry> restrictionBiomes;
 	public boolean spawnInDungeons;
 
+	public boolean spawnInRaids;

```

Comment:
I'd remove the space between `spawnInDungeons` and `spawnInRaids`

Reply:
Done